src/command/player/index.js

```diff
@@ -4,105 +4,151 @@ import emoji from 'node-emoji';
 import playerInfo from './info';
 import seasonStats from './seasonStats';
 
+import playerInfoCompare from './infoCompare';
+
 import NBA from '../../utils/nba';
 import catchAPIError from '../../utils/catchAPIError';
+import seasonStatsCompare from './seasonStatsCompare';
 
 const player = async (playerName, option) => {
   await NBA.updatePlayers();
 
-  const _players = await NBA.searchPlayers(playerName);
-
-  pMap(
-    _players,
-    async _player => {
-      let commonPlayerInfo;
-      let playerHeadlineStats;
-
-      try {
-        const {
-          commonPlayerInfo: _commonPlayerInfo,
-          playerHeadlineStats: _playerHeadlineStats,
-        } = await NBA.playerInfo({
-          PlayerID: _player.playerId,
-        });
-
-        commonPlayerInfo = _commonPlayerInfo;
-        playerHeadlineStats = _playerHeadlineStats;
-      } catch (err) {
-        catchAPIError(err, 'NBA.playerInfo()');
-      }
-
-      if (option.info) {
-        playerInfo({ ...commonPlayerInfo[0], ...playerHeadlineStats[0] });
-      }
+  // split the input by commas and then find matching players and add to array
+  // works even if only input 1 player
+  const _players = [];
+  const nameArray = playerName.split(',');
+  nameArray.forEach(name => {
+    const playerData = NBA.searchPlayers(name.trim());
+    _players.push(...playerData);
+  });
+
+  // if we are comparing we need to get all the data first before make the table
+  if (option.compare) {
+    // do this a little bit different than the original
+    // get all the player data in an array, then will pass to playerInfoCompare and parse it
+    const infoMapper = elem => NBA.playerInfo({ PlayerID: elem.playerId });
+    let playerDataArr;
+    await pMap(_players, infoMapper)
+      .then(result => {
+        playerDataArr = result;
+      })
+      .catch(err => catchAPIError(err, 'NBA.playerInfo()'));
+
+    if (option.info) {
+      playerInfoCompare(playerDataArr);
+    }
+    if (option.regular || option.playoffs) {
+      // if getting regular season data need to get more data
+      const profileMapper = elem =>
+        NBA.playerProfile({ PlayerID: elem.playerId });
+      let playerProfileArr;
+      await pMap(_players, profileMapper)
```

Comment:
The same as [line 31](https://github.com/xxhomey19/nba-go/pull/69/files#diff-2a9d5f44f9b2b46ae219eaded1482a50R31)